docs/djdknativedigest.md

```diff
@@ -26,6 +26,8 @@
 
 This option enables or disables OpenSSL native cryptographic support for the Digest algorithm.
 
+<i class="fa fa-exclamation-triangle" aria-hidden="true"></i> **Restriction:** ![Start of content that applies to Java 8 (LTS)](cr/java8.png) ![Start of content that applies to Java 11 (LTS)](cr/java11.png) Due to issue [#5611](https://github.com/eclipse/openj9/issues/5611), the Digest algorithm is currently disabled. ![End of content that applies only to Java 8 and 11 (LTS)](cr/java_close_lts.png)
+
```

Comment:
Should the default in the chart be changed? It is somewhat misleading.

Reply:
I was assuming that the default applied to JDK 12? Is that not true?

Reply:
It is true the default applies to JDK 12, but it's not clear the table only applies to JDK 12. Also we expect very few people to be reading the docs for JDK 12 as the primary users are JDK 8 and 11.

Reply:
I added an icon in the table. Does that help?
See staging: https://eclipse.github.io/openj9-docs-staging/271/djdknativedigest/

Reply:
Yes. Should `false` also have a checkmark and a tag for jdk 8 and 11?

Reply:
I assume for 8 & 11 the option has no effect. So would it be misleading to suggest that there is a default?

Reply:
I guess. There is a setting, which is false, and it cannot be modified.

Reply:
Let me know if you are done, and I will merge.